### DISCUSSION TITLE
Nginx 400 when i activate X-Forwarded-Host in nginx site conf

### DISCUSSION THREAD

**Discussion by AnneWielis** [netbox-community/netbox#6462] [Q&A] [answered]
### NetBox version

latest (just installed anew)

### Python version

3.8

### Steps to Reproduce

The Line from documentation:
`proxy_set_header` X-Forwarded-Host $http_host;
leads to getting an Error 400 from nginx on every page

### Expected Behavior

Should just work ;-)

### Observed Behavior

Bad Request (400)

**Comment by AnneWielis** [comment, marked as answer]
Hm...so we have a discussion now?

**Reply by candlerb** [reply]
This config works for everyone else, so I have high confidence that the documentation is right.  This suggests something else is different about your system.

It would be a good idea if you could provide:
* the full content of the error message, not just the code 400.  In particular, this will identify whether it's Netbox or Nginx which is generating the error, and the content may give a clue as to the underlying problem
* the full nginx config you are running
* any logs generated by nginx and netbox when this problem occurs

In the absence of any of this information, if I had to guess I'd say it could be a problem with the `ALLOWED_HOSTS` setting in configuration.py.

**Reply by AnneWielis** [reply]
...and your guess was 100% right. After finding out why netbox wasn't logging (for the records: found it at https://github.com/netbox-community/netbox/discussions/5514 ), netbox was complaing that the used hostname was not in allowed_hosts - obvously the IP Address was not sufficient.

Thanks so much for helping!
